CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * [Fix] Fixed CI issue by updating `invalid_connection_string_duckdb` in `test_magic.py` (#631)
 
 * [Fix] Refactored `ResultSet` to lazy loading (#470)
+* [Fix] Fixed Integration tests command getting stuck when database driver is not available. ([#644](https://github.com/ploomber/jupysql/issues/644))
```

Comment:
Please mention the issue number without the link

e.g.

```
- [Fix] Fixes an error that caused function `do_something` to break when passing `0` as input (#99)
```

see more in the [contributing guide](https://github.com/ploomber/contributing/blob/main/contributing/pr.md#changelog)